README.md

```diff
@@ -1,25 +1,43 @@
-  - Frontend should allow users to signup, create accounts and create a room
-  - On room creation, they can share the link with their friend who can join the group as well
-  - Add move validation to make sure users can only make valid moves
-  - Use either canvas or raw HTML blocks for the game. You may also use a game engine like phaser.
-  - You will need to do socket programming since this is a realtime game.
-  
- ### 2 player flappy bird
-  - Create a multiplayer flappy bird game where users can invite their friends and play against each other
-  - Use raw canvas for rendering, and write all physics yourself (Basic physics like acc, velocity ...)
-  - You will need to do socket programming since this is a realtime game. Sockets would relay the positions of users to each other.
-  
- ### Create a UI on top of ChatGPT Api
-  - Create a ChatGPT clone using their APIs
-  - Use their APIs on the backend, relay the information to your frontend
-  - Make sure your frontend experience is 10x better than the native chat GPT experience.
-  - Allow users to put in their own Chat GPT API keys and use GPT 4.
-
- ### Create an multi chain airdropping website 
-   - Build a website that lets users airdrop various crypto currencies to them on their respective testnets
-   - User should be allowed to select the chain (ETH, SOL, Polygon, ARB) and put in an address where they want the native token sent
-   - User should be able to select the amount they want to 
+<br>
+
+![logo](https://raw.githubusercontent.com/itxsoumya/images/main/GSoC.png)
+
+<br>
+
+|  | ♟️ CHESS ♟️ |
+| :----: | :---: |
+| 01 | Build a multiplayer chess game like https://chess.com |
+| 02 | Frontend should allow users to signup, create accounts and create a room |
+| 03 | On room creation, they can share the link with their friend who can join the group as well
+| 04 | Add move validation to make sure users can only make valid moves
+| 05 | Use either canvas or raw HTML blocks for the game. You may also use a game engine like phaser. |
+| 06 | You will need to do socket programming since this is a realtime game. |
+
+<br>
+<br> 
+
+|  | 🐤 PLAYER FLAPPY BIRD 🐤 |
+| :----: | :---: |
+| 01 | Create a multiplayer flappy bird game where users can invite their friends and play against each other |
+| 02 | Use raw canvas for rendering, and write all physics yourself (Basic physics like acc, velocity ...) |
+| 03 | You will need to do socket programming since this is a realtime game. Sockets would relay the positions of users to each other. |
+
+<br>
+<br>
+
+|  | 🤖 CREATE A UI ON TOP OF CHAT-GPT API 🤖 |
+| :----: | :---: |
+| 01 | Create a ChatGPT clone using their APIs |
+| 02 | Use their APIs on the backend, relay the information to your frontend |
+| 03 | Make sure your frontend experience is 10x better than the native chat GPT experience. |
+| 04 | Allow users to put in their own Chat GPT API keys and use GPT 4. |
+
+<br><br>
+
+
+|  | 🔗 CREATE AN MULTI CHAIN AIRDROPPING WEBSITE 🔗 |
```

Comment:
|     | 🔗 CREATE A MULTICHAIN AIRDROPPING WEBSITE 🔗 |

Reply:
thanks for noticing it.
mistakes have been resolved.